packages/ai/cypress/specs/AINoticeIndicator.cy.tsx

```diff
@@ -1,109 +1,162 @@
 import AINoticeIndicator from "../../src/AINoticeIndicator.js";
+import {
+	AI_NOTICE_INDICATOR_CLOSE_BUTTON_TEXT,
+	AI_NOTICE_INDICATOR_POPOVER_CONTENT,
+	AI_NOTICE_INDICATOR_ATTRIBUTIONTEXT,
+	AI_NOTICE_INDICATOR_VERIFICATIONTEXT
+} from "../../src/generated/i18n/i18n-defaults.js";
 
 describe("AI Notice Indicator", () => {
 	describe("Rendering and Interaction", () => {
 		beforeEach(() => {
 			cy.mount(<AINoticeIndicator></AINoticeIndicator>);
 		});
 
-		const attributionText = "Created with AI.";
-		const verificationText = "Verify results before use.";
-
 		it('Should render the component', () => {
 			cy.get('ui5-ai-notice-indicator').should('exist');
 		});
 
-		it("Should display closed responsive poppover", () => {
+		it("Should display closed responsive popover", () => {
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-responsive-popover]")
 				.should("exist");
-
-			cy.get("[ui5-ai-notice-indicator]")
-				.shadow()
-				.find("[ui5-responsive-popover]")
-				.should("have.prop", "open", false);
 		});
```

Comment:
I meant remove the whole `"Should display closed responsive popover"` test, and insted inside the tests where you interact with opening/closing the popover, just add `.should("exist");` after you get it/before other assertions you test on it.